delfin/api/schemas/storage_capabilities_schema.py

```diff
@@ -196,160 +188,161 @@
                                                 'maxLength': 255}
                             },
                         },
-                        'readResponseTime': {
+                        'readThroughput': {
                             'type': 'object',
                             'properties': {
-                                'unit': {'type': 'string', 'enum': ["ms"]},
+                                'unit': {'type': 'string', 'enum': ["MB/s"]},
                                 'description': {'type': 'string',
                                                 'minLength': 1,
                                                 'maxLength': 255}
                             },
                         },
-                        'writeResponseTime': {
+                        'writeThroughput': {
                             'type': 'object',
                             'properties': {
-                                'unit': {'type': 'string', 'enum': ["ms"]},
+                                'unit': {'type': 'string', 'enum': ["MB/s"]},
                                 'description': {'type': 'string',
                                                 'minLength': 1,
                                                 'maxLength': 255}
                             },
                         },
-                        'readThroughput': {
+                        'readIops': {
                             'type': 'object',
                             'properties': {
-                                'unit': {'type': 'string', 'enum': ["MB/s"]},
+                                'unit': {'type': 'string', 'enum': ["IOPS"]},
                                 'description': {'type': 'string',
                                                 'minLength': 1,
                                                 'maxLength': 255}
                             },
                         },
-                        'writeThroughput': {
+                        'writeIops': {
                             'type': 'object',
                             'properties': {
-                                'unit': {'type': 'string', 'enum': ["MB/s"]},
+                                'unit': {'type': 'string', 'enum': ["IOPS"]},
                                 'description': {'type': 'string',
                                                 'minLength': 1,
                                                 'maxLength': 255}
                             },
                         },
-                        'readRequests': {
+                        'cacheHitRatio': {
                             'type': 'object',
                             'properties': {
-                                'unit': {'type': 'string', 'enum': ["IOPS"]},
+                                'unit': {'type': 'string', 'enum': ["%"]},
```

Comment:
Similar comment, can the ratio be anything except %

Reply:
same as above